QwMockData Root Guide

Mrc1104 opened the pull request:
Adds the QwMockData ROOT Guide as a markdown file.

Please send any feedback to Ryan Conaway (mrc@jlab.org).

wdconinc commented on Guide/Images/BCM_example.png:
Text blocks should not be uploaded as pngs. These are not searchable and hard to update by others.

Mrc1104 replied:
I removed the majority of the images and replaced them with inline text. Please take a look and let me know if there is anything else you would like to see changed or added.

paulmking commented on Guide/Guide.md:
```diff
@@ -0,0 +1,802 @@
+
+        $  build/qwmockdatagenerator -r 4 -e 1:20000 --config qwparity_simple.conf \
+           --detectors mock_newdets.map --data <Data>
+        $  build/qwparity -r 4 --config qwparity_simple.conf \
+           --detectors mock_newdets.map --data-handler mock_datahandlers.map \
+           --data <Data> --rootfiles <Rootfiles>
```
Please correct the "--data-handler" flag to be "--datahandlers".

Mrc1104 replied:
I corrected the typo in the --datahandlers argument.

Mrc1104 commented:
In commit a8414df, I corrected the qwparity --datahandlers argument typo. I also added a section on how to do basic plotting of the root trees and their branches. This includes adding a friend between two trees in two different root files. 

Please review and let me know of any feedback.
